Notebooks/DC2_Postage_Stamps.ipynb

```diff
@@ -4,7 +4,9 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## DC2: Generate Postage Stamps for set of RA, Dec coordinates\n",
+    "# DC2: Generate Postage Stamps for set of RA, Dec coordinates\n",
+    "<br>Owner: **Michael Wood-Vasey** ([@wmwv](https://github.com/LSSTDESC/DC2_Repo/issues/new?body=@wmwv))\n",
+    "<br>Last Run: **2018-07-15**\n",
```

Comment:
I'm suggesting the slightly wordier: "Last Verified to Run" to imply a specific process of sign-off.  These numbers will get out of date themselves, and adding the quantifier of sign-off perhaps better conveys the idea that this should run, it was possibly run by someone last week, but we only signed-off that it definitely ran the last time we merged it to master which was on the "Last Verified to Run" date.